dipy/viz/actor.py

```diff
@@ -0,0 +1,648 @@
+
+        def opacity(self, value):
+            if vtk.VTK_MAJOR_VERSION <= 5:
+                self.SetOpacity(value)
+            else:
+                self.GetProperty().SetOpacity(value)
+
+        def copy(self):
+            im_actor = ImageActor()
+            im_actor.input_connection(self.output)
+            im_actor.SetDisplayExtent(*self.GetDisplayExtent())
+            im_actor.opacity(opacity)
+            return im_actor
+
+        def set_position(self, wx, wy, wz):
+            self.SetPosition(wx, wy, wz)
+
+        def get_position(self):
+            return self.GetPosition()
+
+    image_actor = ImageActor()
+    if nb_components == 1:
+        plane_colors = vtk.vtkImageMapToColors()
+        plane_colors.SetLookupTable(lut)
+        plane_colors.SetInputConnection(image_resliced.GetOutputPort())
+        plane_colors.Update()
+        image_actor.input_connection(plane_colors)
+    else:
+        image_actor.input_connection(image_resliced)
+    image_actor.display()
+    image_actor.opacity(opacity)
+
+    return image_actor
+
+
+def streamtube(lines, colors=None, opacity=1, linewidth=0.01, tube_sides=9,
+               lod=True, lod_points=10 ** 4, lod_points_size=3,
+               spline_subdiv=None, lookup_colormap=None):
+    """ Uses streamtubes to visualize polylines
+
+    Parameters
+    ----------
+    lines : list
+        list of N curves represented as 2D ndarrays
+
+    colors : array (N, 3), list of arrays, tuple (3,), array (K,), None
+        If None then a standard orientation colormap is used for every line.
+        If one tuple of color is used. Then all streamlines will have the same
+        colour.
+        If an array (N, 3) is given, where N is equal to the number of lines.
+        Then every line is coloured with a different RGB color.
+        If a list of RGB arrays is given then every point of every line takes
+        a different color.
+        If an array (K, ) is given, where K is the number of points of all
+        lines then these are considered as the values to be used by the
+        colormap.
+        If an array (L, ) is given, where L is the number of streamlines then
+        these are considered as the values to be used by the colormap per
+        streamline.
+        If an array (X, Y, Z) or (X, Y, Z, 3) is given then the values for the
+        colormap are interpolated automatically using trilinear interpolation.
+
+    opacity : float
+    linewidth : float
+    tube_sides : int
+    lod : bool
+        use vtkLODActor rather than vtkActor
+    lod_points : int
+        number of points to be used when LOD is in effect
+    lod_points_size : int
+        size of points when lod is in effect
+    spline_subdiv : int
+        number of splines subdivision to smooth streamtubes
+    lookup_colormap : bool
+        add a default lookup table to the colormap
+
+    Examples
+    --------
+    >>> from dipy.viz import fvtk
+    >>> r=fvtk.ren()
```

Comment:
It looks like that API is implemented.  Perhaps update all the examples to use it?